appveyor.yml

```diff
@@ -3,15 +3,14 @@ version: "{build}"
 os: Windows Server 2012 R2
 
 install:
-  - appveyor DownloadFile https://atom.io/download/windows -FileName AtomSetup.exe
```

Comment:
Yeah, I wanted the stupid Windows build to stop failing. Turns out it's the package installer that's flakey, and by association the Chocolatey install also (i.e. it didn't really help).